contrib/install.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+
+set -e
+
+fail () {
+  >&2 echo
+  >&2 echo "Install failed: $1"
+  >&2 echo
+  >&2 echo "Please contact support@pganalyze.com for help and include information about your platform"
+  exit 1
+}
+
+user_input=''
+yum_opts=''
+apt_opts=''
+pgags_opts=''
```

Comment:
Do we need to add explicit support for `PGA_API_BASEURL` somewhere here? (it looks like `PGA_API_KEY` makes it through, but I couldn't set the base URL programatically, e.g. to test on staging)

Reply:
Sure, I can add that.